timely_beliefs/tests/test_df_resampling.py

```diff
@@ -267,3 +267,13 @@ def test_downsample_once_upsample_once_around_dst(
     )
     assert df_resampled_2.sensor == df.sensor
     assert df_resampled_2.event_resolution == event_resolution_2
+
+
+def test_groupby_preserves_metadata(df_4323: BeliefsDataFrame):
```

Comment:
Is this out of scope for this PR? (Is there a better PR this could go into")

Reply:
Sorry, I worked off of the `specify-support-of-pandas-versions` branch for this PR (because that first created the `dev` folder), and should have set the PR to merge with that branch.